apps/sms/views/shared/test/unit/utils_test.js

```diff
@@ -1153,6 +1153,131 @@ suite('Utils', function() {
     });
   });
 
+  suite('Utils.throttle', function() {
+    test('right typeof parameters', function() {
+      assert.throw(() => Utils.throttle(null), 
```

Comment:
nit: also the minor convention issue: https://github.com/mozilla-b2g/gaia/pull/31677/files#r38888049